pythonwars/util.py

```diff
@@ -12,6 +14,15 @@ def __init__(self, json_data):
             tree = key.split("/")
             leaf = tree.pop()
             look_in = json_data
+            if ":" in leaf:
+                leaf, class_name = leaf.split(":")
+            else:
+                class_name = None
             for inner_key in tree:
                 look_in = look_in[inner_key]
-            setattr(self, inflection.underscore(leaf), look_in[leaf])
+            result = look_in[leaf]
+            if class_name is not None:
+                *module_name, class_name = class_name.split(".")
+                module_name = ".".join(module_name)
+                result = sys.modules[module_name].__dict__[class_name](result)
```

Comment:
This is just crazy. `sys.modules` and attribute tricks (`__dict__` or `getattr`) just for a very basic attribute holder class?